src/components/carouselItem.tsx

```diff
@@ -8,15 +8,17 @@ export interface CarouselItemInfo {
 }
 
 interface CarouselItemProps {
-  info: CarouselItemInfo;
+  text: string;
+  index?: number;
```

Comment:
you don't use index in the component so it can be removed

Reply:
Done